src/actions/users.js

```diff
@@ -236,10 +237,11 @@ export function loadMe(): ActionFunc {
             dispatch(getAllCustomEmojis());
         }
 
-        await Promise.all(promises);
+        const results: $Subtype<ActionResult>[] = await Promise.all(promises);
 
         const {currentUserId} = getState().entities.users;
         Client4.setUserId(currentUserId);
+        Client4.setUserRoles(results[0].data.roles);
```

Comment:
Probably here is a better idea to follow the same pattern than used in setUserId, something like:
```suggestion
        const user = getState().entities.users.profiles[currentUserId];
        Client4.setUserRoles(user.roles);
```

Reply:
:+1: